sidecar/internal/service/reclaimspace.go

```diff
@@ -88,14 +130,25 @@ func (rs *ReclaimSpaceServer) NodeReclaimSpace(
 	pvName := req.GetPvName()
 	klog.Info(pvName)
 
-	// TODO: get following params using kubeClient and pvName
+	pv, err := rs.kubeClient.CoreV1().PersistentVolumes().Get(context.TODO(), pvName, metav1.GetOptions{})
+	volID := pv.Spec.PersistentVolumeSource.CSI.VolumeHandle
+
+	deviceSecret, err := getSecret(rs.kubeClient,
+		pv.Spec.CSI.NodeStageSecretRef.Namespace,
+		pv.Spec.CSI.NodeStageSecretRef.Name)
+
+	var stagingPath string = "/var/lib/kubelet/plugins/kubernetes.io/csi/pv/"
```

Comment:
Make this configurable through cmdline args similar to the following 
https://github.com/ceph/ceph-csi/blob/bc8ef898112169b15b8a749d1f4f6b4d84284680/deploy/rbd/kubernetes/csi-rbdplugin.yaml#L57
https://github.com/ceph/ceph-csi/blob/bc8ef898112169b15b8a749d1f4f6b4d84284680/cmd/cephcsi.go#L67

And add it as a member of ReclaimSpaceServer struct.